app/models/vimeo_channel_details.rb

```diff
@@ -0,0 +1,31 @@
+class VimeoChannelDetails < BaseChannelDetails
+  has_paper_trail
+
+  VIMEO_PREFIX = "vimeo#channel:".freeze
+
+  validates :vimeo_channel_id, presence: true
+  validates :thumbnail_url, presence: true
+  validates :name, presence: true
+  validates :channel_url, presence: true
+
+  def channel_identifier
+    "#{VIMEO_PREFIX}#{vimeo_channel_id}"
+  end
+
+  def omniauth_url
+    "https://api.vimeo.com/oauth/authorize?response_type=code&client_id=7ca223a095b8d2b2aecb9d1317b5af7b5c4c70f3&redirect_uri=https://localhost:3000/publishers/home&state=1"
```

Comment:
We'll want to move this to a ENV variable, but that can be done later

Reply:
I'll be following up with devops to get this account setup